akka-docs/rst/java/code/docs/http/javadsl/server/directives/FutureDirectivesExamplesTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2016-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package docs.http.javadsl.server.directives;
+
+import java.util.concurrent.CompletableFuture;
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.javadsl.server.Marshaller;
+import akka.http.javadsl.server.PathMatchers;
+import akka.http.javadsl.server.Route;
+import akka.http.javadsl.testkit.JUnitRouteTest;
+import akka.http.scaladsl.model.StatusCodes;
+import akka.japi.pf.PFBuilder;
+import org.junit.Test;
+
+import static scala.compat.java8.JFunction.func;
+
+public class FutureDirectivesExamplesTest extends JUnitRouteTest {
+
+  @Test
+  public void testOnComplete() {
+    //#onComplete
+    // import static scala.compat.java8.JFunction.func;
+
+    final Route route = path(
+      PathMatchers.segment("divide")
```

Comment:
Could we static import `PathMatchers.*` and make the path matching a single nice line?